hive/4.0/setup/setup-testdb.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+set -ex pipefail
+
+echo "Waiting for HiveServer2..."
+
+while true; do
+  count=$(grep -cF "Hive Session ID =" "/tmp/hive/hive.log" 2>/dev/null || echo 0)
+  if [ "$count" -ge 2 ]; then
+    break
+  fi
+  sleep 5
+done
+
+echo "HiveServer2 is ready to connect!"
+
+
+HIVE_TEST_DB=testdb
```

Comment:
Great! Can we make it as rewritable env variable? 

As the examaple take a look https://github.com/DataGrip/docker-hive/blob/ba20580f07525351d962193cea548d8f9145f6cd/init.sh#L3